firestore/angular-rewrite/firestore.indexes.json

```diff
@@ -0,0 +1,205 @@
+{
+  "indexes": [
```

Comment:
note: These aren't actually used by the emulator but may be necessary for queries in production